website/content/partials/telemetry-metrics/vault/audit/log_request_failure.mdx

```diff
@@ -0,0 +1,14 @@
+### vault.audit.log_request_failure ((#vault-audit-log_request_failure))
+
+Metric type | Value   | Description
+----------- | ------- | -----------
+counter     | number  | Number of audit log request failures across all devices
+
+The number of request failures is a **crucial metric**.
+
+A non-zero value for `vault.audit.log_request_failure` indicates that all your
+configured audit devices failed to log a request (or response). Vault stops all
```

Comment:
There's still this "Vault stops all users operations when logging fails", which I think is misleading, because it implies that after a logging failure has occurred, we stay in this state where user operations won't run.  I would rather we explain that each request is conditional on auditing completing successfully, e.g.: if all configured audit devices fail to log a request (or response), the request will fail.

Reply:
Got it. It's the failing request that stops, not all Vault functionality.

Reply:
fixed 🤞🏽